docs/getting-started.md

```diff
@@ -68,6 +67,16 @@ The server is accessible via web console at:
     https://192.168.42.10:8443
 ```
 
+Now set your `PATH` to include the version of the `oc` tool bundled with Minishift.  You may already have `oc` installed separately, but this approach releases you from this requirement and additionally ensures that the versions between client and server are aligned.
+
+```console
+$ eval $(minishift oc-env)
+$ which oc
+~/.minishift/cache/oc/v3.10.0/linux/oc
```

Comment:
Why do we have to add this / add instructions with regards to minishift here? I thought that it was as simply as using `minishift start`. Adding these instructions makes it seem a lot more complex than it is... Minishift will automatically download the correct `oc`, so why should we do this manually?